x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ssl/SSLService.java

```diff
@@ -111,47 +121,47 @@
      * Multiple "context names" may map to the same configuration (either by object-identity or by object-equality).
      * For example "xpack.http.ssl" may exist as a name in this map and have the global ssl configuration as a value
      */
-    private final Map<String, SSLConfiguration> sslConfigurations;
+    private final Map<String, SslConfiguration> sslConfigurations;
```

Comment:
Most of the lines-of-code changes in this file are `s/SSLConfiguration/SslConfiguration/` and dropping `Environment` parameters.

I'll comment on the interesting changes.